lib/get-values-from-ci.js

```diff
@@ -0,0 +1,15 @@
+const _ = require('lodash')
+
+const integrationImports = [
+  require('./ci-integrations/travis-ci'),
```

Comment:
Just realized, if we switch to having `test`s in a single file and requiring the `getValue`s on demand this is obsolete anyways.